src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
- [ ] Dear, let's make sure all the linters are enabled. These are here to help us write clean code.